bliss/surveys/dc2.py

```diff
@@ -157,13 +159,13 @@ def setup(self, stage="fit"):
         self.test = self.dc2_data[train_len + val_len :]
 
     def train_dataloader(self):
-        return DataLoader(self.data, batch_size=1)
+        return DataLoader(self.data, shuffle=True, batch_size=128)
```

Comment:
Better to store `batch_size` in the config file, like how it's done in CachedSimulatedDataset:
https://github.com/prob-ml/bliss/blob/417127e4cada1198403f63cc4ed705fbc24b82f0/bliss/simulator/simulated_dataset.py#L258